semantic/src/site/globals/site.overrides

```diff
@@ -2841,9 +2843,15 @@ h6:first-child {
     }
   }
 
-  #notes-tab .options-row {
-    flex-direction: column;
-    align-items: flex-start;
+  #notes-tab {
+    padding: 5px 0px 0px 5px;
+    .options-row {
+      flex-direction: column;
+      align-items: flex-start;      
+    }
+    .item-container {
+      width: 100%;
+    }
   }
```

Comment:
class names get confusing in this area, but hopefully this aligns and spaces things out better in notes tab

![notes_tab_spacing_fix](https://github.com/msupply-foundation/conforma-web-app/assets/17306152/70cbd190-3e64-48d1-911a-ab54ccbd73ec)